### DISCUSSION TITLE
Map has no data

### DISCUSSION THREAD

**Discussion by mauricepot** [flobz/psa_car_controller#547] [General]
Hi,

Map is not working for me. There is no data / route. Can i find the longitude / latitude lag data somewhere?
Added; I see the "trips" (coordinaties) are visible in the comsumptionview. See attached MP4

https://user-images.githubusercontent.com/129339220/236765968-2c1eefcf-1851-49bd-bd52-93fbb7bd801a.mp4

**Comment by Ilpo55** [comment]
It looks that what you see are not cordinates, they are Average Speed and Consumption values.

**Reply by mauricepot** [reply]
OK, clear. But the 'Map" tab does not show driven routes.
